deck/models.py

```diff
@@ -210,8 +213,7 @@ def user_already_voted(self, user):
 
     def user_can_vote(self, user):
         can_vote = False
-        if self.user_already_voted(user) or \
-           (self.author == user and not self.event.author == user):
+        if self.author == user and not self.event.author == user:
```

Comment:
I agree, but I did not touch it because in the PR #180 (yet not merged) this method was refactored. So I'm kind of waiting what is merged first :grin: